pkg/minishift/docker/image/oci_image_handler.go

```diff
@@ -331,21 +334,58 @@ func (handler *OciImageHandler) exportImage(image string, config *ImageCacheConf
 		return fmt.Errorf("Invalid image source '%s': %v", image, err)
 	}
 
-	destRef, err := layout.NewReference(config.HostCacheDir, image)
+	// ImageIndexLocation should be a directory location which will be atomic for each image.
+	// for an image "openshift/origin-control-plane:v3.10.0"
+	// it will be $HOME/.minishift/cache/image/openshift-origin-control-plane-v3.10.0
+	r := strings.NewReplacer(":", "-", "/", "-")
+	ImageIndexLocation := filepath.Join(config.HostCacheDir, r.Replace(image))
+
+	if _, err := os.Stat(ImageIndexLocation); !os.IsNotExist(err) {
```

Comment:
`IsExist`... please do not double negate

Reply:
@gbraad it's not like this there is major difference when to use `IsExist` and when `IsNotExist`, have a look to https://gist.github.com/mastef/05f46d3ab2f5ed6a6787

Reply:
It is about readability...

`not not exists` means `exists`. you also avoid double negativity in a spoken language when possible.

Reply:
confusing logic in golang... sounds like a bad api decision >;-/